app/views/hub/clients/show.html.erb

```diff
@@ -114,10 +114,77 @@
       </div>
     </div>
 
+    <div id="bai">
+      <div class="spacing-below-35">
+        <h2 class="text--small text--bold"><%= "Bank Account Info" %>
+          <span>
+            <% if @client.bank_account_info? %>
+              <%= @show_bank_account_info ? link_to("Hide", hub_client_path(anchor: "bai")) : link_to("View", hub_client_path(bai: true, anchor: "bai"))  %>
+            <% end %>
+          </span>
+        </h2>
+        <% if @show_bank_account_info && @client.bank_account_info? %>
+          <p>
+            <span class="hc-label"><%= "Bank Name" %>:</span>
+            <span class="label-value"><%= @client.intake.bank_name %> </span>
+          </p>
+          <p>
+            <span class="hc-label"><%= "Bank Account Type" %>:</span>
+            <span class="label-value"><%= @client.intake.bank_account_type == "unfilled" ? t("general.NA") : @client.intake.bank_account_type %> </span>
+          </p>
+          <p>
+            <span class="hc-label"><%= "Bank Account Number" %>:</span>
+            <span class="label-value"><%= @client.intake.bank_account_number %> </span>
+          </p>
+          <p>
+            <span class="hc-label"><%= "Bank Routing Number" %>:</span>
+            <span class="label-value"><%= @client.intake.bank_routing_number %> </span>
+          </p>
+        <% elsif @client.bank_account_info? %>
+          <p>
+            <span class="hc-label"><%= "Bank Name" %>:</span>
+            <span class="label-value">●●●●●●●●●● </span>
+          </p>
+          <p>
+            <span class="hc-label"><%= "Account Type" %>:</span>
+            <span class="label-value">●●●●●●●●●●●</span>
+          </p>
+          <p>
+            <span class="hc-label"><%= "Account Number" %>:</span>
+            <span class="label-value">●●●●●●●●●●●</span>
+          </p>
+          <p>
+            <span class="hc-label"><%= "Routing Number" %>:</span>
+            <span class="label-value">●●●●●●●●●●●</span>
+          </p>
+        <% else %>
+          <span class="label-value">No bank account info provided.</span>
+        <% end %>
+      </div>
+    </div>
+
     <div>
       <%= link_to(t("general.edit_info"), edit_hub_client_path, class: "button hc-button--wide") %>
     </div>
   </div>
 
+
   <%= render "client_take_action_footer" %>
+
 <% end %>
+
+<% content_for :script do %>
+  <script>
+      if (window.location.search.includes("bai")) {
```

Comment:
Not really important since we're not using this branch, but using `URLSearchParams` would help here, so we don't accidentally match all words that contain  "bai" (e.g. "bail" or "bailey").

https://developer.mozilla.org/en-US/docs/Web/API/Location/search

Reply:
I have another PR coming up soon for some search params stuff where it can be super simplified using URLSearchParams! Thanks for the tip :D